packages/main/src/Option.ts

```diff
@@ -17,10 +38,16 @@ import type { IOption } from "./Select.js";
  * @extends UI5Element
  * @implements {IOption}
  * @public
- * @abstract
  */
-@customElement("ui5-option")
-class Option extends UI5Element implements IOption {
+@customElement({
+	tag: "ui5-option",
```

Comment:
why do we extend CustomListItem class and at the same time we use the component of the template?